modular_splurt/code/modules/resize/resizing.dm

```diff
@@ -0,0 +1,15 @@
+/mob/living/handle_micro_bump_helping(mob/living/target)
+	// If the target is not a pushover, stop the interaction.
+	if(!HAS_TRAIT(target, TRAIT_PUSHOVER))
+		return FALSE
+
+	// Do the rest of the function normally.
+	. = ..()
+
+/mob/living/handle_micro_bump_other(mob/living/target) // I am not sure why this is a different function.
+	// If the target is not a pushover, stop the interaction.
+	if(!HAS_TRAIT(target, TRAIT_PUSHOVER))
+		return FALSE
+
+	// Do the rest of the function normally.
+	. = ..()
```

Comment:
Please make it so that these checks only run for when it's either harm or grab intents, since those are the ones being complained about